build.ps1

```diff
@@ -1,11 +1,11 @@
 param(
-    [ValidateSet("vs2013", "vs2012", "nupkg", "nupkg-only")]
-    [Parameter(Position = 0)] 
-    [string] $Target = "nupkg",
+    [ValidateSet("vs2013", "vs2015", "nupkg-only")]
```

Comment:
The `libcef_dll_wrapper` was only compiled with `vs2012` and `vs2013`, so targeting `vs2015` is unlikely to work.